### DISCUSSION TITLE
Hi Friends - Offer to share knowledge/collab

### DISCUSSION THREAD

**Discussion by JakeRuth** [OpenBB-finance/OpenBB#1623] [General]
Hi all! I am the cofounder of Stock Unlock. I am a coding nerd, live in NYC, and literally love stonks.... so much. All the love for the stonks. I have been software engineer for a decade mainly in full stack consumer facing web app space

Someone just sent me to this repo and what you're building is freaking awesome. I have spent the last 12+ months tearing through a plethora of different financial data apis (the good and the bad) and have a lot of insider knowledge built up I'd love to share on S&P Global, Bloomberg data, Finnhub, EODHistoricalAPI, ALphaVantage, IEX Cloud, ... I could keep going but I'll stop. 

I am sending this cause I am freaking passionate about disrupting the existing and old financial industry so hard that they have no idea the rug was even pulled out from under them. I see project like this and ours as great opportunities for us to network/stay friendly as we all build the next generation of consumer fintech that actually is incentivized to teach us shit and not push us to make more trades with confetti/preying on users so they can report to their public market bros that they added another 11% profit per user, at their expense since they are pushed to trade without any tools/knowledge to actually equip us with education to make good decisions. Cause like hey, they aren't incentivized to do that, so they won't.

I'll stop ranting, but yea hi! I'd love to chat/help out and share learnings so we can all keep building great shit

- Jake Ruth (Co Founder of Stock Unlock)

**Comment by DidierRLopes** [comment]
Yooo @JakeRuth ,

I've actually seen a TikTok from you last year about creating Stock Unlock! Very nice and impressive work with it! 

My story can be found in [here](https://openbb.co/blog/gme-didnt-take-me-to-the-moon-but-gamestonk-terminal-did).

Let's have a catchup 🚀 🦋

**Reply by JakeRuth** [reply]
I am loving this article!! And wow Tiktok is wild, that's so cool!

"_...I told my girlfriend that on 24 of February (2021) I would make the project public on GitHub, and from then on I would chill down a bit. What a lie…_" - LOL classic

"_...rumour has it that James almost became single that day_" >.< this is crazy

"_In the following weeks I was bringing Naval Ravikant, Elad Gil and Ram Shriram as angels to raise a $8.5 Million round._" - Wow nice!! That's a big win. If you're able to warm intro I am in the middle of a seed round right now/just got out of YCombinator, no expectations though ^_^ we are similar but I am not convinced we are going after the same audience which may remove competition concerns with investors >.> (we are not going after bloomberg terminal users)

"_makes us believe we’ve got a chance against the 100-billion-dollar financial information market dominated by Bloomberg LP, Reuters Eikon, FactSet, S&P Capital IQ, and Morningstar Direct._" - Are you going to be selling financial data? We are paying for Finnhub.io right now but would be interested in being your customer depending on if you offer any b2b data feeds :eyes:

If you're interested a lot of our story is on our HN post which can be found here: https://news.ycombinator.com/item?id=30640295 

Lastly once we finish our seed and I can breathe (lol) I'm down to setup a networking chat! I love connecting with other like minded entrepreneurs and see us as the people who can over throw the older incumbents in our space and I believe there's room for a handful of different platforms/solutions for a nice functioning market :) woo!

**Reply by DidierRLopes** [reply]
Hey @JakeRuth,

We don't sell any financial data because we, OpenBB, don't have any. What we do is we have access to multiple data sources from our platform, effectively connecting data (processed and raw) to users.

Your story is amazing! I enjoyed reading it, but the TikTok was definitely more engaging! 😄 Keep up the good work!

Let's schedule a call for when you are less swamped!

Thanks,
Didier

**Reply by JakeRuth** [reply]
Sounds great, my email is Jake (at) stockunlock (dot) com 🚀

If your investors are interested I can send out blurb/am raising a seed. If not no worries 🙌 and yeah TikTok is the goat for creating engaging content I agree

That's also awesome u source the data yourself that's pretty badass!
